components/workExperience/CompanyLogo.tsx

```diff
@@ -0,0 +1,44 @@
+import Image from "next/image";
+
+import { ExperienceType } from "@/types";
+
+const CompanyLogo = ({ company }: { company: ExperienceType }) => {
+  // NOTE - Regarding TailwindCSS dynamic classes, it cannot be exportable.
+  const imageSources = [
+    {
+      src: company?.logo?.url,
+      className: `block group-hover:hidden dark:hidden dark:group-hover:hidden`,
+    },
+    {
+      src: company?.logo?.srcLightHover?.url,
+      className: `hidden group-hover:block dark:hidden dark:group-hover:hidden group-hover:dark:hidden`,
+    },
+    {
+      src: company?.logo?.srcDark?.url,
+      className: `hidden group-hover:hidden dark:block`,
+    },
+    {
+      src: company?.logo?.srcDarkHover?.url,
```

Comment:
You could use the grayscale tailwind class to make something black and white and then colored instead of having different Hover images, if that's what you're doing here.

Reply:
Thanks for the feedback. I've decided to use brightness-0 to get the exact black colour instead of grayscale.

Reply:
That's great :)